cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -438,12 +438,22 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     profileQuery(
       title = "Empty Result",
       text =
-        """Eagerly loads everything coming in to the `EmptyResult` operator and discards it.""".stripMargin,
+        """The `EmptyResult` operator eagerly loads everything coming in to the `EmptyResult` operator and discards it xxx.""".stripMargin,
       queryText = """CREATE (:Person)""",
       assertions = (p) => assertThat(p.executionPlanDescription().toString, containsString("EmptyResult"))
     )
   }
 
+  @Test def produceResult() {
+    profileQuery(
+      title = "Produce Result",
+      text =
+        """The `ProduceResult` operator ....""".stripMargin,
```

Comment:
The `ProduceResult` operator prepares the result in a way that is consumable by the user - transforming internal values to user values. It's present in every single query that returns data to the user, and not something that usually shows up during performance optimisation.

Not sure into how much detail we should go in here...

Reply:
👍